crates/tests/standalone/build.rs

```diff
@@ -47,6 +47,14 @@ fn main() {
         ],
     );
 
+    // Ensures that root items that reference architecture independent types
+    // of the same name correctly collect all of those types rather than just
+    // the first one
+    write_sys(
+        "src/b_arch_dependencies.rs",
```

Comment:
Be sure to include module in compilation e.g. `mod b_arch_dependencies;` in `lib.rs`.